### DISCUSSION TITLE
[General] ZkSync Era Sepolia Testnet Deployment

### DISCUSSION THREAD

**Discussion by tim-becker** [zkSync-Community-Hub/zksync-developers#220] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

Hi team, I'm curious when zkSync Era plans to transition the testnet to Sepolia considering Goerli will be retired soon?

Other L2s (e.g. Optimism, Base, Arbitrum, etc) have already deployed Sepolia versions of their testnet.

This feature is crucial for developers to test integrations.

### What is the specific issue or error you're encountering?

ZkSync Era Testnet is deployed only on Goerli, which is being retired soon. Testing integrations with other L1 projects is difficult as these projects are moving their testnet deployments to Sepolia.

### Can you share the error messages or logs you're receiving, if any?

N/A

### Have you made any recent changes to the contract before encountering this issue?

N/A

### Are there any external libraries or contracts that your contract interacts with?

N/A

### Can you provide the relevant portions of your contract code where the issue is occurring?

N/A

### Have you tried to isolate the problem, and if so, what were the results?

N/A

### What steps have you already taken to try to resolve the issue?

N/A

**Comment by AlbionaHoti** [comment, marked as answer]
Hi @tim-becker, we're currently working toward migrating to Sepolia testnet for zkSync. Please follow us on our [Developer Twitter account](https://twitter.com/zkSyncDevs) to get the latest developer news for zkSync.

**Reply by MexicanAce** [reply]
https://github.com/zkSync-Community-Hub/zkync-developers/discussions/228

**Comment by keegansa** [comment]
As of now, zkSync Era has not officially announced a transition of their testnet to Sepolia. However, it's advisable to stay updated with their official communications channels, such as their website, blog, or social media, for any announcements regarding the transition. In the meantime, developers can continue testing integrations with zkSync Era on the existing Goerli testnet.

**Comment by robbmoser19** [comment]
The transition of zkSync Era Testnet to Sepolia is essential for developers to test integrations seamlessly with other Layer 1 projects.

**Comment by meutrandeesni** [comment]
As of my last update, there hasn't been any specific announcement regarding the transition of zkSync Era Testnet to Sepolia. Please stay tuned to zkSync's official channels for updates on the testnet transition schedule.

**Comment by porbotufibr** [comment]
zkSync Era team is actively working on transitioning the testnet to Sepolia. Stay tuned for updates on the timeline for this transition.

**Comment by swelexdewi** [comment]
The user is inquiring about when zkSync Era plans to transition their testnet to Sepolia, as other L2s have already deployed Sepolia versions of their testnet. The user is facing difficulty testing integrations with other L1 projects as they are moving their testnet deployments to Sepolia. The user has not provided any error messages or logs, nor have they made any recent changes to their contract. They have not provided any relevant portions of their contract code, nor have they tried to isolate the problem. The user has not provided a repository link and has not provided any additional details.

**Comment by sanddethe** [comment]
The user is asking when zkSync Era plans to transition their testnet to Sepolia, as other L2s have already deployed Sepolia versions of their testnets. The user notes that this feature is crucial for developers to test integrations, and that testing integrations with other L1 projects is difficult as these projects are moving their testnet deployments to Sepolia.
